### ISSUE TITLE
Add Arcsine Distribution

### ISSUE THREAD

**Issue by JimmyMAndersson** [JimmyMAndersson/StatKit#69]
https://en.wikipedia.org/wiki/Arcsine_distribution

**Comment by JimmyMAndersson** [comment]
Putting this on hold since the standard Arcsine distribution is a special case of the Beta distribution. Once the Beta distribution is in place, we may not need this at all (or could perhaps include a convenience function to create it).
